libraries/AP_HAL_ChibiOS/hwdef/common/chibios_board.mk

```diff
@@ -133,6 +133,7 @@ CSRC += $(HWDEF)/common/stubs.c \
 	   $(HWDEF)/common/usbcfg.c \
 	   $(HWDEF)/common/usbcfg_dualcdc.c \
 	   $(HWDEF)/common/usbcfg_common.c \
+      $(HWDEF)/common/usbcfg_cdc_msd.c \
```

Comment:
Can you get the formatting correct? I cleaned this up with my last PR

Reply:
there's still tabs in this file though, I have made changes to be same, but I don't think its still clean.